acceptance/chaos_test.go

```diff
@@ -418,7 +418,8 @@ func testNodeRestartInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfi
 	log.Infof("%d %.1f/sec", count, float64(count)/elapsed.Seconds())
 	kvClient, kvStopper := c.NewClient(t, num-1)
 	defer kvStopper.Stop()
-	if err := kvClient.CheckConsistency(keys.TableDataMin, keys.TableDataMax); err != nil {
-		log.Fatal(err)
```

Comment:
you should leave this as `t.Fatal`, I think. (note that it's currently `log.Fatal`, which is probably wrong)